include/arc_utilities/arc_helpers.hpp

```diff
@@ -1310,14 +1215,7 @@ namespace arc_helpers
             // Init Values Used in Inequality of Interest
             const double val1 = (2 * sqrt(exp(1))) / (lower_bound + sqrt(pow(lower_bound, 2) + 4));
             const double val2 = exp((pow(lower_bound, 2) - lower_bound * sqrt(pow(lower_bound, 2) + 4)) / (4));
-            if (upper_bound > lower_bound + val1 * val2)
-            {
-                return true;
-            }
-            else
-            {
-                return false;
-            }
+            return upper_bound > lower_bound + val1 * val2;
```

Comment:
I like these types of changes. I'm ambivalent about eliding the else statements elsewhere.